File: android/expoview/src/main/java/host/exp/exponent/kernel/services/sensors/BarometerKernelService.java
// Copyright 2015-present 650 Industries. All rights reserved.

package host.exp.exponent.kernel.services.sensors;

import android.content.Context;
import android.hardware.Sensor;

public class BarometerKernelService extends SubscribableSensorKernelService {
  public BarometerKernelService(Context reactContext) {
    super(reactContext);
  }

  @Override
  int getSensorType() {
    return Sensor.TYPE_PRESSURE;
  }
}

File: android/expoview/src/main/java/host/exp/exponent/kernel/services/sensors/BarometerKernelService.kt
// Copyright 2015-present 650 Industries. All rights reserved.
package host.exp.exponent.kernel.services.sensors

import android.content.Context
import android.hardware.Sensor

class BarometerKernelService(reactContext: Context) : SubscribableSensorKernelService(reactContext) {
  override val sensorType = Sensor.TYPE_PRESSURE
}
